uploader/templates/base/base.html

```diff
@@ -3,12 +3,13 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+
     <meta
       http-equiv="Content-Security-Policy"
       content="default-src 'self' https://* http://*;
           img-src 'self' https://*;
           style-src 'self' 'unsafe-inline';
-          script-src 'self' 'unsafe-inline';
+          script-src 'self' 'unsafe-inline' https://* http://*;
```

Comment:
this is scary stuff tbh. I think we have to fix this properly at some point. Specially by implementing proper CORS at the backend.

Reply:
We can specify specific domains as well. The problem was these lines. https://github.com/tonythomas01/gdrive-to-commons/blob/5df6b9917ebc6e157831c39f00feb73340d57986/uploader/templates/upload.html#L239
https://github.com/tonythomas01/gdrive-to-commons/blob/5df6b9917ebc6e157831c39f00feb73340d57986/uploader/templates/upload.html#L292

We can add specific domain names for the time being.  We can use [django-cors-headers](https://pypi.org/project/django-cors-headers/) to add CORS, that can be a separate issuse IMO.